candidate/server.py

```
@@ -1,23 +1,53 @@
+    обрабатывает результат через process_json и возвращает его.
+    """
+    # 1) Сохраняем файл в временную папку
     try:
-        data = await request.read()
+        suffix = os.path.splitext(resume_file.filename)[1]
+        with tempfile.NamedTemporaryFile(delete=False, suffix=suffix) as tmp:
+            temp_path = tmp.name
+            content = await resume_file.read()
+            tmp.write(content)
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Не удалось сохранить файл: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)